README.md

```diff
@@ -7,23 +7,21 @@
 This logo was created for the Tram-One Project, it was designed by [Daniel Jurman](https://danieljurman.com/).
 This version is built off of a single SVG. It uses svgexport to build png variants.
 
-Additionally, you may import this as a javascript module. This will give you access to the png and svg files in your project.
+Additionally, you may import this as a javascript module. This will give you access to the png files, svg files, or svg element in your project.
 
 ## Usage
 
 You can use any of the assets in your projects by directly refrencing the assets from npm.
 
-You can get the svg by pulling directly from the root, or an image from pulling from a specific png found in `dist`:
+### SVG
```

Comment:
a lot of the README was re-organized, so your best bet is to [read it in its entirety](https://github.com/Tram-One/tram-logo/blob/export-element/README.md)